### DISCUSSION TITLE
Fan Speeds can be increased but not fully controlled

### DISCUSSION THREAD

**Discussion by Eeshen** [Rem0o/FanControl.Releases#779] [General]
Hello,

I have ROG Strix X570, I'm noticing a couple things, which I can't figure out.

I have 9 fans, two sets of 3 fans are hooked together and connected to different headers on the MB.  One header I can control the fans from zero to max (WATER_Pump), another header (CPU_OPT) I can increase the speed but cannot fully control the fans (they will only decrease to a certain extent), other headers  (CHA_2, CHA_3) causes the fans to speed up (ramp up) then come to a complete stop, then ramp up again and continues to cycle.  
Other than the water pump, the Chassis headers are all rated at 1amp which is more than my combination of three fans (I'm using Corsair QL fans)

The last three fans (on AIO) are hooked to a Commander Pro but I wasn't expect to control those.  I will be getting another three fan coupler to attach my last three fans to the MB today, but I'm limited to the number of headers I can use because the "cycling" issue.

I was just wondering if anyone had any ideas as to why this is happening.

Lastly, is there a way to start from scratch in Fan Control, like wipeout all my previous setting and start from a clean slate?

Thank you for a great program !!

**Comment by xhul-dev** [comment]
> Lastly, is there a way to start from scratch in Fan Control, like wipeout all my previous setting and start from a clean slate?

Yes there is:
1) Open Fan Control, and in the options, be sure that "Start with Windows" is unchecked.
2) Exit Fan Control completely (right click on tray icon, select "Exit").
3) Delete FanControl folder.
4) Download again.

**Reply by Rem0o** [reply]
Simpler: Delete all *.json files, and the cache file, start fancontrol.

**Comment by Eeshen** [comment]
Going back to my fan control issue.

In another experiment, I have my fans set to 70% Flat curve and they run for about 5 secs then drop to almost off then ramp back to 70% for about 5 secs, then drop again....and the cycle repeats.

I don't understand why, any suggestions where to look?

Using two sets of 3 Corsair QL fans connected to MB.



When I turn off FanControl the "cycling" goes away.

**Reply by xhul-dev** [reply]
> I don't understand why, any suggestions where to look?

How about:
Disable any curve in BIOS (that is, use a fixed value).
Close any software that either controls your fans, and|or monitors them (other than Fan Control).

**Comment by Eeshen** [comment]
Yeah, it appears to have something to do with Q-fan in ASUS MB.   Problem seems to go away when I max the fan speed but then my computer sounds like it is going to take off into outer space on startup and shutdown, until Fan Control kicks in.

Any other thoughts?

**Comment by xhul-dev** [comment]
Can't you use a value lower than 100%?

**Comment by Eeshen** [comment]
I've tried different values, which I notice seems to affect when the cycling kicks in.  At 100% the cycling does not appear to happen as often, until my Mix curve requires the fans to start ramping up.

I lowered the values in Bios and also switched from PWM to DC (just for testing) and when my MIX curve is at 30%, my fans are cycling between full stop and 30%.

**Comment by Eeshen** [comment]
Any other ideas, I turned off all Monitoring by the BIOS, but the fan cycling continues.
